nautobot/circuits/api/serializers.py

```diff
@@ -25,6 +25,7 @@
     NestedCircuitTerminationSerializer,
     NestedCircuitTypeSerializer,
     NestedProviderSerializer,
+    NestedProviderNetworkSerializer,
```

Comment:
Alphabetization nitpick

```suggestion
    NestedProviderNetworkSerializer,
    NestedProviderSerializer,
```